addons/gdUnit4/runtest.cmd

```diff
@@ -16,9 +16,9 @@ IF "%GODOT_TYPE%" == "mono" (
 	ECHO done %errorlevel%
 )
 
-%GODOT_BIN% -s -d .\addons\gdUnit4\bin\GdUnitCmdTool.gd %*
+%GODOT_BIN% --headless -s -d .\addons\gdUnit4\bin\GdUnitCmdTool.gd %*
 SET exit_code=%errorlevel%
-%GODOT_BIN% --no-window --quiet -s -d .\addons\gdUnit4\bin\GdUnitCopyLog.gd %*
+%GODOT_BIN% --headless --quiet -s -d .\addons\gdUnit4\bin\GdUnitCopyLog.gd %*
```

Comment:
good catch, I overlooked this obsolete parameter.

Reply:
I've removed the `--headless` parameter for the first command

Reply:
can you please anwser the questions on the issue https://github.com/MikeSchulze/gdUnit4/issues/249